addons/Hierarchical_Table_Of_Contents/test/ModelValidationTests.js

```diff
@@ -0,0 +1,45 @@
+TestCase("[Hierarchical Table Of Contents] Model validation", {
+    setUp: function () {
+        this.presenter = AddonHierarchical_Table_Of_Contents_create();
+
+        this.model = {
+            ID: "Hierarchical_Table_Of_Contents1",
+            isValid: "True",
+            IsVisible: "True",
+            displayOnlyChapters: "false",
+            expandDepth: "",
+            height: "275",
+            width: "300",
+            titleLabel: "Table of Contents",
+            langTag: "",
+            showPages: "",
+        };
+    },
+
+    'test depth of expand empty': function () {
```

Comment:
```suggestion
    'test given model with depth of expand empty when validating then sets depth of expand to 0': function () {
```

Reply:
Mogę to zmienić w sumie. Znowu, ten kod był skopiowany z innego addonu gdzie występuje to property

Reply:
Te testy które przekopiowałeś są z 2014 roku. Zgodnie z "boy-scout" rule, uważam, że lepiej je uaktualnić.